configure.ac

```diff
@@ -179,6 +193,23 @@ AS_IF([test "x$enable_integration" = xyes],
              [AC_MSG_ERROR([Integration tests require the ss executable])])
        AS_IF([test "x$enable_tctienvvar" != xyes],
              [AC_MSG_ERROR([Integration tests require building with TCTI environment variable support])])
+       AC_SUBST([INTEGRATION_ARGS], [$integration_args])
+      ])
+
+# Integration test with physical device
+AS_IF([test "x$enable_integration" = xyes &&  test "x$with_device_set" = xyes ],
+      [integration_args="$with_device"
+       AC_CHECK_PROG([realpath], [realpath], [yes])
+       AS_IF([test "x$realpath" != xyes],
+             [AC_MSG_ERROR([Integration tests require the realpath executable])])
+       AC_CHECK_PROG([ss], [ss], [yes])
```

Comment:
The `AC_CHECK_PROG` for `ss`  can be removed here, it is only needed to check whether starting the TPM simulator worked.